internal/mode/static/manager.go

```diff
@@ -173,6 +175,19 @@ func StartManager(cfg config.Config) error {
 		LeaderElectionEnabled:    cfg.LeaderElection.Enabled,
 	})
 
+	var usageSecret usage.SecretStore
+	if cfg.UsageReportConfig.SecretNsName != nil {
```

Comment:
My brain is confused by this order. I know that if the secret nsname is not nil, then we must be running plus, but that requires knowledge of how the flags are validated. 

I suggest flipping the order:

```
if cfg.Plus {
  if cfg.UsageReportConfig.SecretNsName != nil {}
  else {}
}
```

Reply:
Sure, that makes sense